src/sagemaker/predictor.py

```diff
@@ -182,10 +182,14 @@ def _create_request_args(
             args["EndpointName"] = self.endpoint_name
 
         if "ContentType" not in args:
-            args["ContentType"] = self.content_type
+            args["ContentType"] = (
+                self.content_type
+                if isinstance(self.content_type, str)
+                else ", ".join(self.content_type)
+            )
 
         if "Accept" not in args:
-            args["Accept"] = ", ".join(self.accept)
+            args["Accept"] = self.accept if isinstance(self.accept, str) else ", ".join(self.accept)
```

Comment:
sanity-check: joining `Accept` and `ContentType` list like this: `type1, type2`? Is the space necessary and is `,` the correct separator?